tools/cargo-compare/src/lib.rs

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) The cargo-guppy Contributors
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use cargo::core::{PackageIdSpec, Workspace};
+
```

Comment:
any reason to separate out this include?

Reply:
just that `merge_imports` isn't stable yet ;)

Reply:
No i mean that there's a space between this `use` statement and the many `use` statements below this line

Reply:
Ah I think IntelliJ did that. Fixed, thanks.